core-java-modules/core-java-exceptions-3/src/test/java/com/baeldung/CopyListUsingAddAllMethodDemoUnitTest.java

```diff
@@ -0,0 +1,19 @@
+package com.baeldung;
+
+import org.junit.jupiter.api.Test;
+
+import java.util.Arrays;
+import java.util.List;
+
+import static com.baeldung.CopyListUsingAddAllMethodDemo.copyList;
+import static org.junit.jupiter.api.Assertions.*;
+
+class CopyListUsingAddAllMethodDemoUnitTest {
+
+    @Test
+    void shouldMakeACopyOfSourceList() {
```

Comment:
Make the test method names follow the givenX_whenY_thenZ convention (givenX is optional).

Reply:
updated the test method names as per convention